src/components/ChatPopUp.tsx

```diff
@@ -71,6 +71,11 @@ export interface ChatPopUpProps
    * CSS classes for customizing the component styling.
    */
   customCssClasses?: ChatPopUpCssClasses;
+  /**
+   * A footer component to render at the bottom of the panel
+   */
+  footer?: React.ReactNode;
+  customHeader?: React.ReactNode;
```

Comment:
footer should be accessible from `Omit<ChatPanelProps, "header" | "customCssClasses">` in this [line](https://github.com/yext/chat-ui-react/pull/41/files#diff-3e73a14d1a8cb05265eccf72131f446caecfabb28c66cea52d29807cbbb0221cR67) so this field is not needed. Similarly for `customHeader`. You can update `Omit<ChatPanelProps, "header" | "customCssClasses">` to `Omit<ChatPanelProps, "customCssClasses">` and access `header` as expected

Reply:
Ah ok!
I updated the code - I exposed the header and removed the `ChatHeader` from `ChatPopUp`. Now the custom header component can be passed to `ChatPopUp`

Reply:
ah yes ok! 
I pushed a fix